doc/_doxygen/groups.dox

```diff
@@ -79,4 +79,14 @@
 @{
 @}
 
+@brief Tests
+@defgroup all_tests Tests
+@{
+@}
+
+@defgroup kernel_memprotect_tests Memory Protection
+@ingroup all_tests
+@{
+@}
```

Comment:
Is there a reason why group is defined in the group.dox and not like the others in the source file?

Reply:
those exist in multiple places and tests, so putting this is a neutral place here.